### ISSUE TITLE
[Bug] Links broken on how to build on various platforms

### ISSUE THREAD

**Issue by djsudduth** [dvorka/mindforger#1478]
**Bug description:**

All the links to 'how to build' on various platforms are broken

**Expected behaviour:**

Go to mindforger.com - scroll to the bottom and select Build: Windows or any other OS and links are broken.  Links also broken in Github - https://github.com/dvorka/mindforger

**Comment by tylerthecoder** [comment]
https://github.com/dvorka/mindforger-repository/commit/a10be29b243f8cc4f147230e4b34cfbba3168778

This commit seems to have broken it

**Comment by Iznik285** [comment]
To add to this the link related to setting up the spell check is also broken. Could you let me know where it is?

**Comment by dvorka** [comment]
@djsudduth @tylerthecoder Apologies! I changed the way how MindForger documentation is written and where it's available - from a GitHub repository to [wiki](https://github.com/dvorka/mindforger/wiki), but I didn't finish it and got pulled to non-MindForger duties.

I published all WIP documentation to https://github.com/dvorka/mindforger/wiki and I also fixed `mindforger` GitHub repository `README.md` links in e6d6dbb70c2e6137737653ab4439e3705c5ecac0

Thank you for :lady_beetle: report!
